starfish/codebook/codebook.py

```diff
@@ -91,34 +68,37 @@ def code_length(self) -> int:
         return int(np.dot(*self.shape[1:]))
 
     @classmethod
-    def _empty_codebook(cls, code_names: Sequence[str], n_channel: int, n_round: int):
-        """create an empty codebook of shape (code_names, n_channel, n_round)
+    def empty(cls, code_names: Sequence[str], n_channel: int, n_round: int):
```

Comment:
I would expect `.empty()` to work like numpy and initialize with whatever values are in memory. May be out of scope for this PR, but would `.zeroes()` be a better method call here?